packages/lit-dev-content/src/components/litdev-playground-share-button.ts

```diff
@@ -115,8 +115,13 @@ export class LitDevPlaygroundShareButton extends LitElement {
   }
 
   override update(changes: PropertyValues) {
-    if (changes.has('_open') && this._open) {
-      this._longUrl?.generateUrl();
+    if (changes.has('_open')) {
+      if (this._open) {
+        this._longUrl?.generateUrl();
+        this.dispatchEvent(new Event('opened'));
```

Comment:
Question: Worth registering event types into the TypeScript event map or making it more specific to the share button?

Reply:
There's no data in this event, and the event map doesn't inform any type checking or anything, so I guess it wouldn't achieve anything practical.

Also `'opened'` is such a generic name that it is likely to collide, and there's no way to declare an event tied to a specific element, right?

Reply:
That's true, maybe prefixing things with the CE name would be a good convention... haven't been doing that generally in this code base, but something to consider as a code improvement generally?